lib/y2lan_restart_common.pm

```diff
@@ -197,6 +266,13 @@ sub check_etc_hosts_update {
     set_network;
 }
 
+=head2 handle_Networkmanager_controlled
+
+ handle_Networkmanager_controlled();
+
+Handle Networkmanager controls the network configurations
```

Comment:
I think here it requires some adjustment, I cannot get it from the description, it seems to be a popup.